nomad/volumewatcher/volumes_watcher_test.go

```diff
@@ -52,8 +52,8 @@ func TestVolumeWatch_EnableDisable(t *testing.T) {
 	require.Equal(0, len(watcher.watchers))
 }
 
-// TestVolumeWatch_Checkpoint tests the checkpointing of progress across
-// leader leader step-up/step-down
+// TestVolumeWatch_LeadershipTransition tests the correct behavior of
+// claim reaping across leader step-up/step-down
 func TestVolumeWatch_Checkpoint(t *testing.T) {
```

Comment:
Should we change the function name to match the comment, or visa versa?

Reply:
Oops, thanks for catching that.